tests/unit/conftest.py

```diff
@@ -0,0 +1,21 @@
+from contextlib import contextmanager
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def switch_region():
```

Comment:
same as previously. `DEFAULT_REGION` should be treated as legacy

Reply:
The unit test for the incoming requests has been deleted (as it doesn't make sense now that we replace the partition with the static default partition). However, for the response partition alignment, the default region is still the fallback for determining the partition if the ARN does not contain a region.